packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -912,7 +912,9 @@ export class DeltaManager
     /**
      * Closes the connection and clears inbound & outbound queues.
      */
-    public close(error?: ICriticalContainerError): void {
+    // back-compat: IDeltaManage.close() has been deprecated in 0.45.
+    // @param reason should eventually become required once definitions are updated and deprecation is completed.
```

Comment:
How does optional first parameter support back compat?  Shouldn't it be second param?